### PR TITLE
Tyler 2141 responsive embed

### PR DESCRIPTION
**Pull request by tylermachado**
Per [helpscout 2141](https://secure.helpscout.net/conversation/605312337/2141?folderId=1847296) --

This should add the PHP to wrap [embed] shortcut output in a new div, as well as the CSS to make that output responsive regardless of what the embedded iframe code demands.

### CODE REVIEW

**Comment by benlk** [comment]
commentary: this appears to merge master into staging. 👍

File: wp-content/themes/freshenergy/functions.php [hunk 1]
```diff
@@ -77,6 +77,17 @@ function category_and_tag_archives( $wp_query ) {
...
 
+/**
+ * oembed is a pain; this fixes fixed widths for responsive output
+ *
+ * Wraps embeds in a container div.
+ */
+function mwe_responsive_embed($html, $url, $attr, $post_ID) {
+	$return = '<div class="video-container">'.$html.'</div>'; return $return;
+}
+add_filter( 'embed_oembed_html', 'mwe_responsive_embed', 10, 4 );
```
Comment (benlk) [inline wp-content/themes/freshenergy/functions.php:88]: This takes 4 parameters, but only uses the `$html`. And [this comment](https://developer.wordpress.org/reference/hooks/embed_oembed_html/#comment-2414) says that the example there, which looks the same as this code, appears to affect *all* URLs, not just YouTube's.

So some questions:

- Can this be limited via $url or $attr or something to only affect YouTube embeds?
- How does this change affect embeds from sites other than YouTube? The full list of oembed providers is here: https://codex.wordpress.org/Embeds At minimum we'd want to check:
    - Instagram
    - Facebook
    - Twitter
    - DocumentCloud

And like @kaylima said, the prefix here should be changed to reflect that this isn't `mwen_`, but is `fe_`

**Review by benlk — CHANGES_REQUESTED** [review]
I think this needs more testing?

**Review by benlk — APPROVED** [review]

File: wp-content/themes/freshenergy/functions.php (hunk 1 above)
Comment (tylermachado) [reply wp-content/themes/freshenergy/functions.php:88]: also update for MWE

**Pull request closed without merging**